glossary.md

```diff
@@ -81,23 +82,33 @@ rating(s) and other fields associated with a `response`. The following specifica
 pertain to rated instances, _not_ `registry` definitions.
 
 * `Link`: Defines which `survey objects` are connected. The rating is applied to that `link`.
-* `Performance rating`: 0-100 rating of the performance of the subject. Answers the question: 
-  What is your satisfaction with this input? (0=No performance, 100 = perfect)
 * `Criticality rating`: 0-10 rating of the criticality of an input to an output,
   e.g. criticality of an `observing system` to a `data product`. This answers the question:
   On a scale of 1-10, how much would the loss of this input impact the performance of your 
   `data product` or `application` (1 - very little impact; 10 - complete loss of performance).
-* `Rationale`: Why a `criticality` or `performance` rating was selected.
+* `Criticality rationale`: Text description answering the question: What accounts for this
```

Comment:
@hazelshapiro looks like there will now be 2 bullets for criticality rationale? did you just want to add this info to the original bullet or replace?

Reply:
Maybe I'm missing the duplicate, but not seeing it. We have `Criticality rating` and `Criticality rationale` as separate fields that look really similar though :)

Reply:
This addresses this issue: https://github.com/nsidc/usaon-vta-docs/issues/9 @hazelshapiro would you agree that this PR resolves that issue fully?

Reply:
@MattF-NSIDC my eyes really need checked today!

Reply:
Yes, I think so! It's hard with so many similar fields. There should be criticality rating / criticality rationale / performance rating / performance rationale.